OSVRUnreal/Plugins/OSVR/Source/OSVR/Private/OSVRHMD.cpp

```diff
@@ -496,6 +504,36 @@ FOSVRHMD::FOSVRHMD()
 	GEngine->bSmoothFrameRate = false;
 }
 
+FOSVRHMD::FOSVRHMD(FOSVRHMD* other)

```

Comment:
I'm not sure we really want to enable copy-construction here (especially manually implementing the copy constructor) unless it's really needed and we're providing value, not reference, semantics.